docs/odh-rhoai/image-puller.md

```diff
@@ -0,0 +1,66 @@
+# Using Kubernetes Image Puller to Speed Up Start-Up Times
```

Comment:
I'd add "Operator" in the title. 

and in the article, I'd mention that it's a **community** operator.